### PR TITLE
Ide 4666

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -16,6 +16,7 @@ Export-Package: com.liferay.blade.gradle.model,
 Require-Bundle: biz.aQute.bndlib;bundle-version="[4,5)",
  com.liferay.ide.core,
  com.liferay.ide.project.core,
+ com.liferay.ide.scripting.core,
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF]: do we need this?

Comment (simonjhy) [reply tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF]: we need groovy lib, can only find in this bundle

Comment (simonjhy) [reply tools/plugins/com.liferay.ide.gradle.core/META-INF/MANIFEST.MF]: removed this

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

**Comment by jtydhr88** [comment]
sent to Greg, thanks.

**Pull request closed without merging**